### DISCUSSION TITLE
Fabric support

### DISCUSSION THREAD

**Discussion by hirbod** [kirillzyusko/react-native-keyboard-controller#7] [Ideas]
Since RN is evolving quickly, new libraries should also support Fabric if possible. Since Fabric won't be a real thing widely in 1-2 years, I think old and new architecture should be supported.

**Comment by kirillzyusko** [comment]
Hi @hirbod 

Yes, I'm planning to add a support for Fabric architecture.

The initial idea was to stabilize the first version of the library using old architecture and implement all requested features. And once it's done, then add support for the Fabric (most likely it will be released as second version of the library 😎)

**Reply by kirillzyusko** [reply]
A PR with basic fabric support: https://github.com/kirillzyusko/react-native-keyboard-controller/pull/64
Most likely it will be merged (after intensive testing) and released as `1.2.0`

**Comment by thanksyouall** [comment]
@kirillzyusko Should we close this ticket?

**Reply by kirillzyusko** [reply]
I think @hirbod already closed it after your comment :)
Thanks for noticing it 🙌
